### PR TITLE
DEVDOCS-2204: Fix getImage Handlebars Helpers

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2204](https://jira.bigcommerce.com/browse/DEVDOCS-2204)

## What changed?
* Added examples, params, and improved description for the following helpers:
    * `getImageManagerImage`
    * `getImageManagerImageSrcset`
    * `getContentImage`
   * `getContentImageSrcset`

### CODE REVIEW

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 1]
```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)
```
Comment (traci707) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:313]: I am curious if the links in the Helper column should also go to github pages like getContentImage. I am wondering why the links are needed in this column.

**Review by traci707 — APPROVED** [review]

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 2]
```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
...
 
-Returns sized image URL from store `/content` directory.
+Returns URL for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getContentImage).
+#### Parameters
+
+* `a` **{String}:** Image path relative to `/dav/content/`
+* `width` **{Int}:** Width in pixels
```
Comment (bookernath) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:319]: Might be worth mentioning that both width and height are optional.

Validation combinations are:
- No height/No width (returns original image)
- Width only
- Width + height

Not valid:
- Height only

This is true for both non-srcset helpers.

Comment (bookernath) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:319]: Never mind, I see this is covered a bit later on.

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 3]
```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)
```
Comment (uwikaiddi) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:312]: @aglensmith, linking these headers does not give context into what kind of information the link is leading to. The links should be separated out similar to the Cornerstone example links.

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md (hunk 3 above)
Comment (ektravel) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:312]: I agree with @uwikaiddi. I think we need to change the format of this article and separate the names of the helpers and the associated links. 
```suggestion
### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 4]
```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
 
 ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)
 
-Returns sized image URL from store `/content` directory.
+Returns URL for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
```
Comment (ektravel) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:314]:
```suggestion
Returns URL for an image uploaded to the WebDAV **content** folder. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 5]
```diff
@@ -311,13 +311,54 @@ Returns language translation keys as JSON string.
...
+<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123x321/content/asset.jpg -->
+
+{{getContentImage "asset.jpg" width=123}}
+<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123w/content/folder/asset.jpg -->
+```
 
 ### [{{getContentImageSrcset}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImageSrcset.js)
 
-Returns an `<img>` tag `srcset` value for images  `/content` directory (accessible via WebDAV). Refer to [getImageSrcSet](#getimagesrcset) for usage examples.
+Returns a `srcset` for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
```
Comment (ektravel) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:343]:
```suggestion
Returns a `srcset` for an image uploaded to `/dav/content/`. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 6]
```diff
@@ -339,18 +380,58 @@ You can use the optional `defaultImage` parameter to specify an image that will
 {{getImage image "thumbnail"}}
 ```
 
-
 [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getImage).
 
 ### [{{getImageManagerImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getImageManagerImage.js)
 
-Returns an `<img>` tag `src` value for images in a store's `/product_images/uploaded_images` directory (accessible via [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV)).
+Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image URL for an image [uploaded to `/dav/product_images/uploaded_images`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
```
Comment (ektravel) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:387]:
```suggestion
Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image URL for an image uploaded to `/dav/product_images/uploaded_images`. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md [hunk 7]
```diff
@@ -339,18 +380,58 @@ You can use the optional `defaultImage` parameter to specify an image that will
...
+{{getImageManagerImage "folder/asset.jpg" width=123 height=321}}
+<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123x321/image-manager/folder/asset.jpg -->
+```
 
 [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getImageManagerImage).
 
 ### [{{getImageManagerImageSrcset}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getImageManagerImageSrcset.js)
 
-Returns `<img>` tag `srcset` value for image's in a store's `/product_images/uploaded_images` directory (accessible via [WebDav](https://support.bigcommerce.com/s/article/File-Access-WebDAV)). Refer to [getImageSrcSet](#getimagesrcset) for usage examples.
+Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image `srcset` for an image [uploaded to `/dav/product_images/uploaded_images`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).
```
Comment (ektravel) [inline docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:417]:
```suggestion
Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image `srcset` for an image uploaded to `/dav/product_images/uploaded_images`.

```

**Review by ektravel — COMMENTED** [review]
I suggest removing WebDAV links from helper definitions and adding a link to File Access (WebDAV)(https://support.bigcommerce.com/s/article/File-Access-WebDAV) in Resources.

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md (hunk 3 above)
Comment (aglensmith) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:312]: @uwikaiddi that's not related to this specific fix that nate requested, so it needs to go into a separate ticket.

File: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md (hunk 1 above)
Comment (aglensmith) [reply docs/stencil-docs/reference-docs/handlebars-helpers-reference.md:313]: This isn't related to the fix nate requested, so it should go in a separate ticket and not hold up this pull request.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**